NetKVM/Common/ParaNdis_Common.cpp

```diff
@@ -341,6 +345,8 @@ static void ReadNicConfiguration(PARANDIS_ADAPTER *pContext, PUCHAR pNewMACAddre
             pContext->RSC.bIPv4SupportedSW = (UCHAR)pConfiguration->RSCIPv4Supported.ulValue;
             pContext->RSC.bIPv6SupportedSW = (UCHAR)pConfiguration->RSCIPv6Supported.ulValue;
 #endif
+            pContext->uMaxFragmentsInOneNB = pConfiguration->MaxFragmentsInOneNB.ulValue;
```

Comment:
Can we do just "pContext->uMaxFragmentsInOneNB = MAX_FRAGMENTS_IN_ONE_NB"?

Reply:
Yes,we can. I misunderstood what you meant before.